query_optimizer/ExecutionGenerator.cpp

```diff
@@ -135,7 +135,7 @@ static const volatile bool join_hashtable_type_dummy
     = gflags::RegisterFlagValidator(&FLAGS_join_hashtable_type,
                                     &ValidateHashTableImplTypeString);
 
-DEFINE_string(aggregate_hashtable_type, "LinearOpenAddressing",
+DEFINE_string(aggregate_hashtable_type, "SeparateChaining",
```

Comment:
Currently changing from `LinearOpenAddressing` to `SeparateChaining` appears to improve the performance of all TPC-H queries that involve aggregation, but I'm not sure about SSB @pateljm.